package.json

```diff
@@ -30,8 +30,9 @@
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.2.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
     "@aurodesignsystem/eslint-config": "^1.3.0",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.3",
```

Comment:
Update `peerDependencies` to reference the new `@aurodesignsystem/webcorestylesheets`